http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -662,6 +676,77 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       }
   } yield ()
 
+  private def getLandingZoneResources(billingProfileId: String, userToken: Authorization)(implicit
+    ev: Ask[F, AppContext]
+  ): F[LandingZoneResources] =
+    for {
+      // Step 1: call LZ for LZ id
+      landingZoneOpt <- wsmDao.getLandingZone(billingProfileId, userToken)
+      landingZone <- F.fromOption(
+        landingZoneOpt,
+        AppCreationException(s"Landing zone not found for billing profile ${billingProfileId}")
+      )
+      landingZoneId = landingZone.landingZoneId
+
+      // Step 2: call LZ for LZ resources
+      lzResourcesByPurpose <- wsmDao.listLandingZoneResourcesByType(landingZoneId, userToken)
+
+      aksClusterName <- getLandingZoneResourceName(lzResourcesByPurpose,
+                                                   "Microsoft.ContainerService/managedClusters",
+                                                   "SHARED_RESOURCE"
+      )
+      batchAccountName <- getLandingZoneResourceName(lzResourcesByPurpose,
+                                                     "Microsoft.Batch/batchAccounts",
+                                                     "SHARED_RESOURCE"
+      )
+      relayNamespace <- getLandingZoneResourceName(lzResourcesByPurpose,
+                                                   "Microsoft.Relay/namespaces",
+                                                   "SHARED_RESOURCE"
+      )
+      storageAccountName <- getLandingZoneResourceName(lzResourcesByPurpose,
+                                                       "Microsoft.Storage/storageAccounts",
+                                                       "SHARED_RESOURCE"
+      )
+      vnetName <- getLandingZoneResourceName(lzResourcesByPurpose,
```

Comment:
For the VNet + 2 subnets we can probably use the name + parent. Ex:
```
 {
          "resourceType": "DeployedSubnet",
          "resourceName": "AKS_SUBNET",
          "resourceParentId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/Microsoft.Network/virtualNetworks/lz77e23c6133c90e3e813cb742a66a88f47fa65d811720e1b4d53ab3092f85ca",
          "region": "eastus2"
        }
```

Reply:
Based on how we're using the AKS_SUBNET name, it looks to me like we just want the resourceName for the subnets (I could add some logic that confirms that the resourceParentId matches the vnet name we have recorded), and then we use the subnetName + vnetName together.